hazelcast/src/main/java/com/hazelcast/map/impl/SizeEstimator.java

```diff
@@ -16,18 +16,101 @@
 
 package com.hazelcast.map.impl;
 
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.nio.UnsafeHelper;
+import sun.misc.Unsafe;
+
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+import javax.management.openmbean.CompositeDataSupport;
+import java.lang.management.ManagementFactory;
+
 /**
  * Size estimator general contract.
  *
  * @param <T> the type of object which's size going to be estimated.
  */
-public interface SizeEstimator<T> {
+public abstract class SizeEstimator<T> {
+
+    public static final boolean HAS_COMPRESSED_OOPS = hasCompressedOops();
+
+    @SuppressWarnings("CheckStyle")
```

Comment:
You can replace the entry in the `suppressions.xml` with `@SuppressWarnings("checkstyle:magicnumber")` here. That should work (must be all lowercase).